### PR TITLE
Unignore test

### LINKED ISSUE #231: Test ignored - test_get_filtered_containers_success

### PR DESCRIPTION
**Pull request by oritstone**
Fixes #231
Signed-off-by: Orit Stone <orit.stone@hp.com>

### CODE REVIEW

**Comment by meirwah** [comment]
Test case failed: testClear_containers_wrapper_Success

**Comment by oritstone** [comment]
@orius123, this PR includes also changing identical names of test cases, so that they will run. On of the test cases changes fails, so it is now ignored. Opened an issue for it: #235

File: test/io/cloudslang/docker/containers/clear_container.inputs.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@
 #   http://www.apache.org/licenses/LICENSE-2.0
 #
 ####################################################
-testDocker_run_container_Success:
+testClear_container_Success:
   testSuites: [docker]
```
Comment (Bonczidai) [inline test/io/cloudslang/docker/containers/clear_container.inputs.yaml]: should add here `containers` test suite, right?

File: test/io/cloudslang/docker/containers/get_filtered_containers.inputs.yaml [hunk 2]
```diff
@@ -7,7 +7,7 @@
 #
 ####################################################
 test_get_filtered_containers_success:
-  testSuites: [ignore]
+  testSuites: [docker]
```
Comment (Bonczidai) [inline test/io/cloudslang/docker/containers/get_filtered_containers.inputs.yaml]: should add here `containers` test suite, right?

**Comment by Bonczidai** [comment]
please pull from master

**Comment by oritstone** [comment]
Done

File: test/io/cloudslang/docker/containers/clear_container.inputs.yaml [hunk 3]
```diff
@@ -6,7 +6,7 @@
 #   http://www.apache.org/licenses/LICENSE-2.0
 #
 ####################################################
-testDocker_run_container_Success:
+testClear_container_Success:
```
Comment (meirwah) [inline test/io/cloudslang/docker/containers/clear_container.inputs.yaml]: container_Success? really?

**Comment by meirwah** [comment]
test fail:
[ERROR] - Test case failed: test_get_filtered_containers_success - Tests SUCCESS case - filters out containers based on busybox:latest image
		Expected result: SUCCESS
		Actual result: RUN_CONTAINER_TOMCAT_PROBLEM

not a circleci issue...

**Comment by Bonczidai** [comment]
build fails because of disk space issue:
```
Error pulling image (latest) from tomcat, Untar exit status 1 write /usr/lib/jvm/java-7-openjdk-amd64/jre/lib/rhino.jar: no space left on device
```

maybe because (as I see) now all the flows in the `containers` folder use `clear_docker_containers` instead of `clear_docker_host` (why? - to reduce build time?)

WDYT we should do? delete also the images before this test (or in all the tests?), maybe find another small image like busybox to use in the test? something else?

**Comment by meirwah** [comment]
try to change to clear_docker_host , and see if it solves the issue.
busybox is pretty small (around ~5MB) so I would not invest in finding smaller image...

**Comment by oritstone** [comment]
@Bonczidai in the containers flows we changed them to use clear_docker_containers in order to save build time. But it seems to be problematic in terms of disk size on travis...
So we gradually change them back.
We should try to fix the disk size issues on travis, and then change it back to make the build faster

**Comment by oritstone** [comment]
@meirwah please review & merge

**Comment by meirwah** [comment]
:+1:

**Pull request merged**